webpack-demo/src/listFunctions.js

```diff
@@ -26,7 +27,15 @@ function displayTasks() {
     const checkbox = document.createElement('input');
     checkbox.type = 'checkbox';
     checkbox.id = 'checkbox';
+    if (toDoArr.completed) {
+      checkbox.setAttribute('checked', true);
+      li.classList.add('completed');
+    }
     li.appendChild(checkbox);
+    checkbox.addEventListener('click', () => {
+      // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Let's not disable linters because they are there to correct errors and make our code follow standards.

Reply:
Hi @elisha2kyakpo1  thanks for pointing it out
can you help me out or suggest what to do to solve it?

Reply:
Addressed